gnat2goto/unit/floating_point_tests.adb

```diff
@@ -34,7 +34,8 @@ package body Floating_Point_Tests is
         & "00000000"
         & "0000000";
    begin
-      pragma Assert (Convert_Ureal_To_Binary_IEEE (Eight) = IEEE_Bits);
+      pragma Assert (Convert_Ureal_To_Binary_IEEE (Eight, 23, 8, 127)
```

Comment:
Should use the 32 bit helper rather than repeating the hardcoded values here

Reply:
There are no helpers for the conversion to binary, because I though people wouldn't want to use them directly, i.e. outside the conversion to Hex. I can add the helpers here as well.

Reply:
Ah I didn't see that. Hm, in that case your call I'd say :woman_shrugging: